react/components/Backdrop.tsx

```diff
@@ -1,23 +1,24 @@
 import React from 'react'
-import { useCssHandles } from 'vtex.css-handles'
-import { TransitionProps } from 'react-transition-group/Transition'
+import type { TransitionProps } from 'react-transition-group/Transition'
+import type { CssHandlesTypes } from 'vtex.css-handles'
 
 import Fade from './Animations/Fade'
 import styles from '../styles.css'
 
 export type BackdropMode = 'display' | 'clickable' | 'none'
 
+export const CSS_HANDLES = ['backdropContainer', 'backdrop'] as const
+
 interface Props {
   open: boolean
   transitionDuration?: TransitionProps['timeout']
   onClick?: (e: React.MouseEvent<HTMLDivElement>) => void
+  handles: CssHandlesTypes.CssHandles<typeof CSS_HANDLES>
 }
 
-export const CSS_HANDLES = ['backdropContainer', 'backdrop'] as const
-
-const Backdrop: React.FC<Props> = props => {
-  const { children, open, onClick, transitionDuration } = props
-  const handles = useCssHandles(CSS_HANDLES)
+const Backdrop: React.FC<Props> = (props) => {
```

Comment:
I saw that some functional components defined as arrow functions were converted to function. This isn't the same case?

```suggestion
function Backdrop(props: Props) {
```

Reply:
Changing!